src/sentry/incidents/logic.py

```diff
@@ -1290,18 +1302,16 @@ def deduplicate_trigger_actions(
     """
     actions = _prioritize_actions(triggers=triggers)
 
-    deduped = {}
+    deduped: dict[tuple[Any, ...], AlertRuleTriggerAction] = {}
```

Comment:
`tuple[Any, ...]` should be replaced with the real type

Reply:
Done: https://github.com/getsentry/sentry/pull/75541/commits/a33de845d89b1bfe4d6ea2f249eb883fab54aa35

I'd just as soon define a dataclass as list out all that stuff, if it weren't a one-off.